_pages/cfp.md

```diff
@@ -139,13 +139,13 @@ Yes, but please don’t submit more than two.  If both your talks are accepted,
 
 Yes! We’d love to have you. Please note that you’ll be responsible for obtaining a visa, should you need one, to enter Canada.
 
-
 **Can I get feedback on my application?**
 
 Unfortunately, we will not be able to offer this. We simply don’t have the resources to offer this to everyone at a quality we feel is appropriate.
 
 **I have other questions! Who do I talk to?**
 
-Our Speaker & CFP Coordinator is Anna Lorimer, send any and all CFP-related questions to her at [uwstarconcfp@gmail.com](mailto:uwstarconcfp@gmail.com).
+Plese send any and all CFP-related questions to our Speakers Team at [starcon-speakers@googlegroups.com](mailto:starcon-speakers@@googlegroups.com).
```

Comment:
There's an extra @ in the address here. At the moment only people in the google group can post to it - I think if we change the post permissions it will be fine but we should probably test this (unless you have already)

Reply:
Good catch! I've updated the post permissions to be anyone on the web (viewing and replying permissions are still restricted to members)

Reply:
thanks team!